backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -67,7 +67,9 @@ public enum GLEmulation {
 
 	int idleFPS = 60;
 	int foregroundFPS = 0;
+	int backgroundFPS = 0;
 
+	boolean activateBackgroundFPS = false;
```

Comment:
This also looks odd and different from how the rest of the Lwjgl3 FPS config works. Normally backgroundFPS would be initialized to some meaningful but invalid "sentinel" number like -1 or -2, and if it still has that value later, then that means it was never set. If it was set, such as to 0 to have unlimited background FPS, 60 to have reasonably high background FPS, or 5 to have very low-power-usage background FPS, then you know it was intentional because it isn't that negative sentinel value.